src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -225,5 +225,9 @@ public static class ScriptConstants
         public static readonly string LiveLogsSessionAIKey = "#AzFuncLiveLogsSessionId";
 
         public static readonly string FunctionsHostingConfigSectionName = "FunctionsHostingConfig";
+
+        // HTTP Proxying constants
+        public static readonly string IsHttpProxyingEnabled = "IsHttpProxying";
+        public static readonly string HttpProxyCorrelationHeader = "x-invocation-id";
```

Comment:
Going by other extension headers, I think we want `x-ms-` as the prefix.
```suggestion
        public static readonly string HttpProxyCorrelationHeader = "x-ms-invocation-id";
```